lib/linguist/languages.yml

```diff
@@ -3298,6 +3298,10 @@ Mathematica:
   - ".wlt"
   aliases:
   - mma
+  - wolfram
+  - wolframlanguage
+  - wolframlang
```

Comment:
It would be better to set these aliases to "wolfram language" and "wolfram lang" as Linguist automatically adds aliases that replace each space with a hyphen and empty string as well: "wolfram lang" will add aliases "wolframlang" and "wolfram-lang" without needing to do that manually. See https://github.com/github/linguist/issues/5135#issuecomment-757868541

Reply:
Added the spaces.